lib/paseto.rb

```diff
@@ -43,13 +43,9 @@ def self.encode64(bin)
     Base64.strict_encode64(bin).gsub(/=+$/, '')
```

Comment:
(I did this) I just want to point out that I think this is a bit of a hack and would love verify actually works with other implementations. Not sure if there is a better way to do it and maintain compat with Ruby 2.3, newer Ruby has a flag to skip padding.